molecule/default/molecule.yml

```diff
@@ -40,6 +40,7 @@ provisioner:
   env:
     CATALINA_HOME: "/opt/apache-tomcat-9.0.50"
     ANSIBLE_FORCE_COLOR: "true"
+    ANSIBLE_VERBOSITY: 3
```

Comment:
Please remove, that for debugging purpose :)

BTW, you can just add Env Var to the build with the same name (ANSIBLE_VERBOSITY) and normally Ansible will use it. (Unless I'm wrong, Ansible always checks if this Env Var is defined, when it starts running).